views/place.hbs

```diff
@@ -1,54 +1,65 @@
+{{>header}}
+{{>options}}
 
-<ul class="">
-  <div class="">
-    <a href="/">{{localText.homePage}}</a>
-    <a href="/{{lang}}/places">{{localText.backList}}</a>
+<a class="f6 ma4 w4 tc grow no-underline br-pill ba bw1 pv2 dib mid-gray b gray" href="/{{lang}}/add-place">{{localText.add}}</a>
+<div class="pa4">
+  <div class="overflow-auto">
+    <table class="f6 w-100 mw8 center bb b--turquoise bw2" cellspacing="0">
+      <thead>
+        <tr class="ma2">
+          <th class="fw6 tl pa3">
+            {{#if place.imageUrl}}
+              <img src="{{place.imageUrl}}" alt="{{localText.image}}">
+            {{else}}
+              <img src="https://upload.wikimedia.org/wikipedia/commons/a/ac/No_image_available.svg" alt="{{localText.image}}">
+            {{/if}}
+          </th>
+          <th class="f1 tc dark-text">{{place.local.name}}</th>
+          <th class="fw6 pa3">
+              <a href="/{{../lang}}/edit-place/{{_id}}" class="f6 w4 grow no-underline br-pill ba bw1 pv2 mv2 db mid-gray b gray">{{localText.edit}}</a>
```

Comment:
should be like before still: https://github.com/foundersandcoders/OTP-Data-Entry/pull/77/files#diff-ce4b2d284ce877aee88b4bbb6e2da75bL47